sequencing/ATACSeq/config/config.txt

```diff
@@ -1,14 +1,39 @@
+## ===========================================================##
+##           ATAC-seq pipeline: Example config file           ##
+## ===========================================================##
+##                                                            ||
+## This is an example config file for the ATAC-sequencing     ||
+## pipeline. This is expected to be located in the project's  ||
+## directory. The full path to this file should be specified  ||
+## when running the pipeline's scripts.                       ||
+##                                                            ||
+## The project's folder name needs to be specified in the     ||
+## first variable. Please specify the modules versions to be  ||
+## used.                                                      ||
+##                                                            ||
+## ===========================================================##
+
 ## Sequencing data filepaths
+export PROJECT="project name"
 export DATADIR=/lustre/projects/Research_Project-MRC190311/ATACSeq/${PROJECT}
```

Comment:
Can we change these to be more generic (i.e remove reference to ISCA file paths)?